README.md

```diff
@@ -49,24 +49,21 @@ Installation of Nexus IQ Server application is to `/opt/sonatype/nexus-iq-server
 
 By default, the IQ Server reads its [main configuration file](https://help.sonatype.com/iqserver/configuring/config.yml) from within the image at `/etc/nexus-iq-server/config.yml`.
 
-There is an environment variable `JAVA_OPTS` that is being used to pass JVM arguments to the java command that launches IQ server. The default value is `-Djava.util.prefs.userRoot=${SONATYPE_WORK}/javaprefs`.
```

Comment:
Seeing that the Dockerfile itself wasn't changed (which makes sense to me), I'm not clear why we had to change this sentence which merely states the still true default value for this env var.

Reply:
> Seeing that the Dockerfile itself wasn't changed (which makes sense to me), I'm not clear why we had to change this sentence which merely states the still true default value for this env var.

Yeah it's not really a needed change, just thought it might be better to condense a bit and move it to the product licensing section since we mention it's defaulted there too